src/com/boringdroid/systemui/AppStateLayout.java

```diff
@@ -76,7 +76,9 @@ private void removeTask(int taskId) {
     private void topTask(ActivityManager.RunningTaskInfo runningTaskInfo) {
         String packageName = runningTaskInfo.baseActivity == null ?
                 null : runningTaskInfo.baseActivity.getPackageName();
-        if (packageName != null && packageName.startsWith("com.farmerbb.taskbar")) {
+        if (packageName != null && ( packageName.startsWith("com.farmerbb.taskbar") ||  
```

Comment:
From here, we should add a method to check Launcher, instead of add another package name to here. The current `isLauncher` methods do this work, but it looks like not work correctly. Maybe we should fix this method to let it work correctly, and remove checking for third-party Launcher with specific package name.

Reply:
I feel it would also be a better idea to fix the isLauncher method vs making more of these types of changes.

Reply:
I decide to merge it, and I will check this problem when I have more time to manage `boringdroid-9.0.0` branch.